docker/docker_guide.md

```diff
@@ -1,69 +0,0 @@
-# Running ESMValTool packaged in the docker container
```

Comment:
I would like to keep this file and have it updated with instructions on how to use the new docker image.

Reply:
DONE